homeassistant/components/tankerkoenig/sensor.py

```diff
@@ -59,7 +60,7 @@
                 )
                 continue
             sensor = FuelPriceSensor(
-                fuel, station, coordinator, f"{NAME}_{station['name']}_{fuel}"
+                fuel, station, coordinator, f"{NAME}_{station['name']}_{fuel}", show
```

Comment:
you could directly use "tankerkoenig.show_on_map" instead of having the variable.
```suggestion
                fuel, station, coordinator, f"{NAME}_{station['name']}_{fuel}", tankerkoenig.show_on_map
```